prijateli_tree/app/routers/games.py

```diff
@@ -339,6 +321,9 @@ def view_round(
     # Get current round
     if current_round == 1:
         template_data["ball"] = player.initial_ball
+    elif current_round > game.rounds:
+        redirect_url = f"/games/{game_id}/player/{player_id}/end_of_game"
```

Comment:
The redirect should be to the function that adds the score.  As a complication, we don't want to add a score for practice games.